src/app/store.ts

```diff
@@ -1,6 +1,9 @@
 import { combineSlices, configureStore } from '@reduxjs/toolkit';
+import { todosSlice } from '../features/todos';
+import { currentTodoSlice } from '../features/currentTodo';
+import { filterSlice } from '../features/filter';
 
-const rootReducer = combineSlices();
+const rootReducer = combineSlices(todosSlice, currentTodoSlice, filterSlice);
```

Comment:
The function `combineSlices` does not exist in `@reduxjs/toolkit`. You should use `combineReducers` instead to combine the slices: `import { combineReducers } from '@reduxjs/toolkit';` and then use `combineReducers` here.